### PR TITLE
docs(ntv): ntv site examples

### PR DESCRIPTION
**Pull request by BBSQQ**
image

-----------
image
image

### CODE REVIEW

File: site/examples/ntv/case/demo/meta.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+  "title": {
+    "zh": "中文分类",
+    "en": "Category"
+  },
+  "demos": [
+    {
+      "filename": "explain-chart.tsx",
+      "title": {
+        "en": "Explain Chart",
```
Comment (pddpd) [inline site/examples/ntv/case/demo/meta.json:10]: Explain is a verb 🤔

File: site/examples/ntv/interactive/demo/meta.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
+{
+  "title": {
+    "zh": "中文分类",
+    "en": "Category"
+  },
+  "demos": [
+    {
+      "filename": "link-chart.tsx",
+      "title": {
+        "en": "Linkage of Chart",
```
Comment (pddpd) [inline site/examples/ntv/interactive/demo/meta.json:10]: 从英文来看是图表联动的意思（另外，可视化术语里的联动通常用 coordinated

File: site/examples/ntv/case/demo/meta.json (hunk 1 above)
Comment (BBSQQ) [reply site/examples/ntv/case/demo/meta.json:10]: How about 'chart description'?

File: site/examples/ntv/interactive/demo/meta.json (hunk 2 above)
Comment (BBSQQ) [reply site/examples/ntv/interactive/demo/meta.json:10]: How about 'coordinated with chart'?

File: site/examples/ntv/case/demo/meta.json (hunk 1 above)
Comment (BBSQQ) [reply site/examples/ntv/case/demo/meta.json]: Rename as 'Chart Explanation'

File: site/examples/ntv/interactive/demo/meta.json (hunk 2 above)
Comment (BBSQQ) [reply site/examples/ntv/interactive/demo/meta.json]: Renamed

File: site/examples/ntv/basic/index.zh.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
+---
+title: NTV 基本用法
```
Comment (chenluli) [inline site/examples/ntv/basic/index.zh.md:2]: NTV 这个缩写名字有在其他地方声明么，用户能 get 到吗，要不文档标题还是写全称？

Comment (chenluli) [reply site/examples/ntv/basic/index.zh.md:2]: done

**Review by chenluli — APPROVED** [review]
LG

**Pull request merged**